### ISSUE TITLE
Docker image fails with http.client.RemoteDisconnected error (README example)

### ISSUE THREAD

**Issue by felciano** [mintapi/mintapi#566]
**Describe the bug**
When running the default Docker example from the README:
```
docker run --rm --shm-size=2g ghcr.io/mintapi/mintapi mintapi john@example.com my_password --headless --use-chromedriver-on-path
```
`mintapi` fails with a `Remote end closed connection without response` error.

**What version of MintAPI are you using?**
Not sure how to tell what version of `mintapi` is in the Docker instance?

**What command[s] did you run / steps to reproduce?**
See above

**What did you expect to happen?**
Successful completion with mint data printed to console.

**What actually happened?**
Exceptioned out.

I suspect this is related to #546 and/or #553. 

I'm not sure how to determine what version of chrome and chromedriver are in the Docker image. Is there a way to determine this and potentially have Docker builds for each of the most recent versions? That way we can back out to previous chrome versions if chrome-specific bugs pop up again.

**Additional context**

Stack trace is here:
```
Mint sync apparently incomplete after timeout.  Data retrieved may not be current.
Traceback (most recent call last):
  File "/home/seluser/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/home/seluser/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 449, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/home/seluser/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 444, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/lib/python3.8/http/client.py", line 1348, in getresponse
    response.begin()
  File "/usr/lib/python3.8/http/client.py", line 316, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.8/http/client.py", line 285, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/seluser/.local/bin/mintapi", line 8, in <module>
    sys.exit(main())
  File "/home/seluser/.local/lib/python3.8/site-packages/mintapi/cli.py", line 518, in main
    data = mint.get_account_data(limit=options.limit)
  File "/home/seluser/.local/lib/python3.8/site-packages/mintapi/api.py", line 262, in get_account_data
    return self.get_data(
  File "/home/seluser/.local/lib/python3.8/site-packages/mintapi/api.py", line 242, in get_data
    data = self.__get_mint_endpoint(endpoint, **kwargs)
  File "/home/seluser/.local/lib/python3.8/site-packages/mintapi/api.py", line 449, in __get_mint_endpoint
    response = self.get(
  File "/home/seluser/.local/lib/python3.8/site-packages/mintapi/api.py", line 162, in get
    return self.driver.request("GET", url, **kwargs)
  File "/home/seluser/.local/lib/python3.8/site-packages/seleniumrequests/request.py", line 159, in request
    self.requests_session.headers = get_webdriver_request_headers(self, proxy_host=self.__proxy_host)
  File "/home/seluser/.local/lib/python3.8/site-packages/seleniumrequests/request.py", line 76, in get_webdriver_request_headers
    webdriver.switch_to.window(original_window_handle)
  File "/home/seluser/.local/lib/python3.8/site-packages/selenium/webdriver/remote/switch_to.py", line 134, in window
    self._w3c_window(window_name)
  File "/home/seluser/.local/lib/python3.8/site-packages/selenium/webdriver/remote/switch_to.py", line 142, in _w3c_window
    send_handle(window_name)
  File "/home/seluser/.local/lib/python3.8/site-packages/selenium/webdriver/remote/switch_to.py", line 138, in send_handle
    self._driver.execute(Command.SWITCH_TO_WINDOW, {'handle': h})
  File "/home/seluser/.local/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py", line 427, in execute
    response = self.command_executor.execute(driver_command, params)
  File "/home/seluser/.local/lib/python3.8/site-packages/selenium/webdriver/remote/remote_connection.py", line 344, in execute
    return self._request(command_info[0], url, body=data)
  File "/home/seluser/.local/lib/python3.8/site-packages/selenium/webdriver/remote/remote_connection.py", line 366, in _request
    response = self._conn.request(method, url, body=body, headers=headers)
  File "/home/seluser/.local/lib/python3.8/site-packages/urllib3/request.py", line 78, in request
    return self.request_encode_body(
  File "/home/seluser/.local/lib/python3.8/site-packages/urllib3/request.py", line 170, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/home/seluser/.local/lib/python3.8/site-packages/urllib3/poolmanager.py", line 376, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/home/seluser/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 787, in urlopen
    retries = retries.increment(
  File "/home/seluser/.local/lib/python3.8/site-packages/urllib3/util/retry.py", line 550, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/home/seluser/.local/lib/python3.8/site-packages/urllib3/packages/six.py", line 769, in reraise
    raise value.with_traceback(tb)
  File "/home/seluser/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/home/seluser/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 449, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/home/seluser/.local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 444, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/lib/python3.8/http/client.py", line 1348, in getresponse
    response.begin()
  File "/usr/lib/python3.8/http/client.py", line 316, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.8/http/client.py", line 285, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
```

**Comment by mpearon** [comment]
I'm having an issue similar to @felciano - called differently, but with the same result.

```
mintapi --headless --no_wait_for_sync --accounts --transactions --include-investment --filename $mintTempDirectory my_username my_password
```

**Comment by apiology** [comment]
https://github.com/cryzed/Selenium-Requests/issues/53 has been fixed and released in selenium-requests 2.0.1. I verified it resolves this issue for me. Run pip install --upgrade mintapi 'selenium-requests>=2.0.1' to upgrade.

**Comment by mrooney** [comment]
I just released mintapi 2.10 to pypi, which updates selenium-requests to 2.0.1. Please try that (or update your selenium-requests to the same) and let me know if that resolves the issue. Thanks!
